package.json

```diff
@@ -11,6 +11,7 @@
     "build": "webpack --mode=production",
     "build:dev": "webpack --mode=development",
     "build:watch": "webpack --watch --mode=development",
+    "size": "size-limit",
```

Comment:
If you run this task just after running the dev build (i.e. you just ran the dev server), you'll get a very different number. I'm wondering if we should:
- include an explicit call to `npm run build` to this script OR 
- add something to the README so that potential adopters can run this command upfront

Reply:
What about making it part of the `test` ci build?

Reply:
Great, I've pushed this change up.